src/components/DescriptionList/DescriptionList.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { Fragment } from 'react';
+
+import { parseHtml } from 'utils/parse-html';
+
+export type DescriptionProps = {
+  title: string;
+  markup: string;
+  url: string;
+  name: string;
+};
+
+type DescriptionListProps = {
+  items: DescriptionProps[];
+  id: string;
+};
+
+export const DescriptionList = ({ items, id }: DescriptionListProps) => (
+  <div className="cc-description-list">
+    <dl>
+      {items
+        .filter(({ markup }) => markup !== null)
+        .map(({ title, markup }) => (
+          <Fragment key={title}>
+            <dt className="cc-description-list__title">{title}</dt>
+            <dd className="cc-description-list__definition">
+              {parseHtml(markup)}
```

Comment:
Personally I would use `RichText` here as it both parses the HTML for us and allows to do some global transforms (e.g. adding selectors, preventing script tags being inserted).

Can understand there may be times not to use this, so fair enough if this is one of those!

Reply:
Hey - I did consider that, but it added a nested `div` which felt unnecessary / didn't read well semantically.

Bearing this in mind, I took the `parseHtml` function which as far as I can tell performs santisation. Would definitely be possible to run this through `formatHTMLString` as well if you think it's worth it?

One thing to bear in mind is that the links need to open in the same window.

Reply:
If it's just the `<div>` element that puts you off using it then I think

Reply:
I don't think we need to worry about an extraneous `<div>` element - the `dd` element can contain any flow content as children (see https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dd), so I don't think we should worry about a div.

Re: links in same window - this is something which needs to change globally, so I would say still use the RichText in this instance because we'll be making a change to RichText soon enough that ensures `<a>` tags with external URLs are no longer automatically marked up with `target="_blank" rel="noopener noreferrer"`.

Reply:
Sure, that makes good sense.

Just so you're aware; there are two types of content that corporate-react is going to place here ..

![image](https://user-images.githubusercontent.com/33448330/107968270-08edf980-6fa6-11eb-8442-0cd134e607c8.png)

* The first type is a list of terms; and can be seen in the top three items in the screenshot.
* The second type is a block of rich text; and can be seen in the bottom two items.

Reply:
Bearing this in mind, I'm wondering whether both should be displayed within a `RichText` block. And if so, perhaps I should be inserting a comma character for list item via corporate-react, rather than adding the comma using a pseudo element?

Reply:
That is helpful thanks.

So actually only the first type is markup, and the second type is a text string (though both ultimately come through as strings?).

My thought is that we could use RichText regardless, and add a className of (e.g.) `u-dis-i` - a utility selector which sets `display: inline`. I think this would work, though could be wrong.